examples/amulware.Graphics.Examples.Basics/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("amulware.Graphics.Examples.Basics")]
```

Comment:
Do we need this file at all?

Reply:
Good catch. This was automatically generated and I forgot to delete it.